### PR TITLE
Repeat value: question mark in main and tests (Add fix link commit)

### PR DESCRIPTION
**Pull request by sunhuachuang**
the value had in file: `question-mark-in-main-and-tests.html`

### CODE REVIEW

**Comment by sunhuachuang** [comment]
And third is also a link in index.html

**Comment by sunhuachuang** [comment]
And fourth is also a link in cargo workspace's page.  : )

File: src/rust-2018/error-handling-and-panics/the-question-mark-operator-for-easier-error-handling.md [hunk 1]
```diff
@@ -120,131 +118,3 @@ case, `?` will return a value for `Some(T)` and return `None` for `None`. One
 current restriction is that you cannot use `?` for both in the same function,
 as the return type needs to match the type you use `?` on. In the future,
 this restriction will be lifed.
-
```
Comment (steveklabnik) [inline src/rust-2018/error-handling-and-panics/the-question-mark-operator-for-easier-error-handling.md]: sorry, why are we removing all of this?

Comment (sunhuachuang) [reply src/rust-2018/error-handling-and-panics/the-question-mark-operator-for-easier-error-handling.md]: Isn't this for that ` ? in main and tests ` section?

**Pull request merged**